R/get_code.R

```diff
@@ -63,26 +70,21 @@ get_parameter_code <- function(.expr, .assgn) {
 # returns as output an expression (or code) without the branch
 compute_branch <- function(.expr, .assgn) {
   assigned_parameter_option_name = .assgn[[.expr[[2]]]]
-  parameter_values = .expr[-1:-2]
-  
-  assigned_parameter_option_value = map(parameter_values, ~ get_option_index_from_branch(.x, assigned_parameter_option_name)) %>% 
-    flatten_lgl() %>%
-    which(arr.ind = TRUE)
-  
-  parameter_values %>%
-    extract2(assigned_parameter_option_value) %>%
-    get_option_value() 
-}
+  option_names = map(.expr[-1:-2], ~ get_option_name(.x))
```

Comment:
`map(value, ~ f(.x))` can be written more simply as `map(value, f)`; i.e. here that's `map(.expr[-1:-2], get_option_name)`